src/registryDAO/types.mligo

```diff
@@ -30,11 +30,14 @@ type config_proposal =
   ; max_proposal_size : nat option
   }
 
+type update_guardian = address
```

Comment:
I guess this could be moved to the common module and used in `treasuryDAO` as well

Reply:
Moved